arrays/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func main() {
+	// Array a can hold 5 integeres.
+	// The type of elements and the type are part of the array's type.
+	// By default an array is zero-valued, which for integer means 0.
+	var a [5]int
```

Comment:
This (arrays in Go) is good knowledge to have, as it's the building blocks of slices, but in backend microservices you'll probably be using slices all the time, and basically never arrays.